### DISCUSSION TITLE
Allow to filter diff output

### DISCUSSION THREAD

**Discussion by Hofer-Julian** [borgbase/vorta#858] [Ideas]
Currently diff outputs all changes, including permission changes and deleted or added files/directories.
For certain situations the output can be overwhelming, so I suggest to add

- [x] added
- [x] deleted
- [x] modified

checkboxes to the diff output dialog to allow fine grained control

**Comment by m3nu** [comment]
I think you already implemented this. So closing it here. @Hofer-Julian

**Comment by Hofer-Julian** [comment]
I did not @m3nu :)
I used the checkboxes for demonstration how it would look like, not for my progress.

**Comment by m3nu** [comment]
Fair enough. I added some labels, since I'm cleaning up issues today.

**Comment by m3nu** [comment]
Still working on this? Which milestone can I assign as target?

**Comment by Hofer-Julian** [comment]
No, sorry I am not working on it

**Reply by m3nu** [reply]
(moved to discussion > feature ideas)
